armsrc/lfops.c

```diff
@@ -478,13 +481,62 @@ static void fc(int c, int *n) {
             dest[((*n)++)]=1;
             dest[((*n)++)]=1;
             dest[((*n)++)]=1;
+            dest[((*n)++)]=1;
+            dest[((*n)++)]=1;
             dest[((*n)++)]=0;
             dest[((*n)++)]=0;
             dest[((*n)++)]=0;
             dest[((*n)++)]=0;
             dest[((*n)++)]=0;
-            dest[((*n)++)]=0;
-            dest[((*n)++)]=0;
+        }
+    }
+}
+// compose fc/X fc/Y waveform (FSKx)
+static void fcAll(uint8_t c, int *n, uint8_t clock, uint16_t *modCnt) 
+{
+    uint8_t *dest = BigBuf_get_addr();
+    uint8_t idx;
+    uint8_t fcCnt;
+    // c = count of field clock for this bit
+    uint8_t mod = clock % c;
+    uint8_t modAdj = c/mod;
+    bool modAdjOk=FALSE;
+    if (c % mod==0) modAdjOk=TRUE;
+    // loop through clock - step field clock
+    for (idx=0; idx < (uint8_t) clock/c; idx++){
```

Comment:
I have old comments that need removing as well.
I'll look at using memset.  That command just doesn't come naturally to me, but loops do.  (I do agree it probably reads simpler to most ppl tho.)